### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by itdaniher**
@transfaeries is working on this branch. It's great. PRing to start discussions.

### CODE REVIEW

**Comment by itdaniher** [comment]
``` bash
pipenv run sh -c 'ADMIN=+15133278483 FULL_SERVICE_URL=http://localhost:9090/wallet SIGNAL_CLI_PATH=./auxin-cli RUST_LOG=debug RUST_BACKTRACE=1 DEBUG=true BOT_NUMBER=+12696666006 NO_MEMFS=true NO_DOWNLOAD=true SIGNAL=auxin $(which python) mobfriend.py'
```

we probably want this invocation (or something like it!) in the README.

**Review by cxloe — APPROVED** [review]
looks really good.

File: README.md [hunk 1]
```diff
@@ -1,76 +1,103 @@
...
-- `ADMIN_METRICS`: send python and roundtrip timedeltas for each command to ADMIN
+For more detailed instructions visit the [Signal-cli repository](https://github.com/AsamK/signal-cli). If your build is failing, first ensure that you're using a version of Java 17 or highuer with java --version
 
-## Other stuff
+## Registering a Signal Account for your bot
 
-Code style: `mypy *py` and `pylint *py` should not have errors when you push. run `black`. prefer verbose, easier to read names over conciser ones.
+You will need at least 2 signal accounts to properly test your bot. A signal account for the bot to run on and your own signal account to talk to the bot. To set up an additional signal account for your bot you can use a second phone or a VoIP service such as Google Voice, [Forest Contact](/contact), Twilio, or Teli.net. All you need is a phone number that can receive SMS.
+
+With a phone number from Google Voice, Forest Contact, Twilio, or Teli.net, a Signal account can be registered easily. These commands (bash compatible) serve as a starting point and use `human-after-all` as an alternative to manually solving the recaptcha challenge.
```
Comment (technillogue) [inline README.md:94]: maybe mention signal-captcha.desktop from https://github.com/forestcontact/message-in-a-bottle/pull/3

File: README.md [hunk 2]
```diff
@@ -1,76 +1,103 @@
...
-- `NO_DOWNLOAD`: don't download a signal-cli datastore, instead use what's in the current working directory
-- `NO_MEMFS`: if this isn't set, MEMFS is started, making a fake filesystem in `./data` and used to upload the signal-cli datastore to the database whenever it is changed. if not `NO_DOWNLOAD`, also create an equivalent tmpdir at /tmp/local-signal, chdir to it, and symlink signal-cli process and avatar
-- `NO_MONITOR_WALLET`: monitor transactions from full-service. relevent only if you're giving users a payment address to send mobilecoin not with signal pay.  has bugs
-- `SIGNAL_CLI_PATH`: executable to use. useful for running graalvm tracing agent
-- `MIGRATE`: run DB migrations (needed when creating a new DB) and set teli sms webhooks
-- `LOGFILES`: create a debug.log
-- `LOGLEVEL`: what log level to use for console logs (DEBUG, INFO, WARNING, ERROR). 
-- `ORDER`: allow users to buy phonenumbers with `/order` and `/pay shibboleth`
-- `GROUPS`: use group routes, allowing `/mkgroup` (aka `/query`), using groups to manage to/from context
-- `ADMIN_METRICS`: send python and roundtrip timedeltas for each command to ADMIN
```
Comment (technillogue) [inline README.md:72]: Most of these flags should still be kept in the top level readme

**Comment by technillogue** [comment]
This should close #52 and probably #51

File: README.md (hunk 1 above)
Comment (transfaeries) [reply README.md]: I think that's a method with too much overhead for the user, since it requires them to install a program.

**Comment by transfaeries** [comment]
This should be ready to merge. @itdaniher please check for conflicts and merge at your convenience.

**Pull request merged**